.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,15 +1,17 @@
-This checklist is used to make sure that common issues in a pull request are addressed. This will expedite the process of getting your pull request merged and avoid extra work on your part to fix issues discovered during the review process. 
+Hello & Welcome to our repo,
 
-### PR information
-- [ ] The title of the PR is clear and informative.
-- [ ] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/cleaning-up-commits.md).
-- [ ] Except for special cases involving multiple contributors, the PR is started from a fork of the main repository, not a branch.
-- [ ] If applicable, the PR references the bug/issue that it fixes.
-- [ ] Swagger files are correctly named (e.g. the `api-version` in the path should match the `api-version` in the spec).
+You Can find a link to our review process [here](https://github.com/Azure/adx-documentation-pr/wiki/swagger-management). if you can't access this link, register your github your account to [Azure Github](https://aka.ms/azuregithub).
 
-### Quality of Swagger
-- [ ] I have read the [contribution guidelines](https://github.com/Azure/azure-rest-api-specs/blob/master/.github/CONTRIBUTING.md).
-- [ ] My spec meets the review criteria:
-  - [ ] The spec conforms to the [Swagger 2.0 specification](https://github.com/OAI/OpenAPI-Specification/blob/master/versions/2.0.md).
-  - [ ] The spec follows the guidelines described in the [Swagger checklist](../documentation/swagger-checklist.md) document.
-  - [ ] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR. 
+**What's happening to your PR now**
+
+The following validation tools are currently running in the following order in the CI and the results will be posted in the travis Job results at the bottom of this page shortly. to run these tools next time on your machine, click on the "Run it locally" link below to see the instructions.
+ |
+ | - > **Semantic Validator** ([Run it locally](https://github.com/Azure/adx-documentation-pr/wiki/Azure-Swagger-Tools#semantic-validator))
+ |
+ | - > **Azure Linter** ([Run it locally](https://github.com/Azure/adx-documentation-pr/wiki/Azure-Swagger-Tools#azure-linter))
+ |
+ | - > **Model Validation** ([Run it locally](https://github.com/Azure/adx-documentation-pr/wiki/Azure-Swagger-Tools#model-validator))
+ | 
+ | - > **Breaking Changes Tool** (Run it locally)
```

Comment:
I've updated the issue for linking to the how to run. 

 | - > **Breaking Changes Tool** ([Run it locally](https://github.com/Azure/adx-documentation-pr/wiki/Azure-Swagger-Tools#breaking-change-validation))